tests/test_fixtures/rule_based_profiler/quentin_user_workflow_verbose_profiler_config.yml

```diff
@@ -13,33 +13,17 @@ variables:
   num_bootstrap_samples: 9139
   bootstrap_random_seed: 43792
 
-  # BatchRequest yielding thirty five (35) batches (January, 2018 -- November, 2020 trip data)
```

Comment:
Love that the configuration is being simplified, but would like to ensure the information in the in-line comment is preserved. What are your thoughts on including the in-line comment whenever this BatchRequest is used in other tests?

Reply:
@Shinnnyshinshin I recall putting this comment next to python version of these batch requests in the tests.